### PR TITLE
[FEATURE] spk setup - create HLD to Manifest pipeline

### LINKED ISSUE #1108: Create HLD to Manifest pipeline automatically
**As a:**
user

**I want:**
create HLD to Manifest pipeline automatically via `spk setup` command

**So that:**
I can see the execution of this pipeline.

**Describe the solution you'd like:**
Add creation in HLD to Manifest pipeline in `spk setup` command

**Acceptance Criteria:**

**Describe alternatives you've considered:**

**Additional context:**

**Does this require updates to documentation?:**
Yes, part of the setup command doc

related to https://github.com/microsoft/bedrock/issues/986

### PR DESCRIPTION
**Pull request by dennisseah**
closes https://github.com/microsoft/bedrock/issues/1108

### CODE REVIEW

**Comment by samiyaakhtar** [comment]
@dennisseah Can you upload a screenshot of what the experience looks like? Helps review :)

**Comment by dennisseah** [comment]
> @dennisseah Can you upload a screenshot of what the experience looks like? Helps review :)

It is a command tool so the screenshot it is going to be pretty simple.

Screen Shot 2020-03-06 at 11 34 47 AM

File: src/lib/setup/pipelineService.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  try {
+    logger.info(`Deleting pipeline ${pipelineName}`);
+    await buildApi.deleteDefinition(projectName, pipelineId);
+  } catch (e) {
+    logger.error(`Error in deleting pipeline ${pipelineName}`);
+    throw e;
+  }
+};
+
+export const getPipelineBuild = async (
```
Comment (samiyaakhtar) [inline src/lib/setup/pipelineService.ts:96]: missing jsdoc

**Comment by dennisseah** [comment]
interactive mode, remains the same that's no new questions.
Screen Shot 2020-03-06 at 11 38 19 AM

**Review by samiyaakhtar — APPROVED** [review]

File: src/lib/setup/pipelineService.ts (hunk 1 above)
Comment (dennisseah) [reply src/lib/setup/pipelineService.ts:96]: accept

**Review by mtarng — APPROVED** [review]

**Pull request merged**